lib/src/Uncertainty/Model/EventRandomVector.cxx

```diff
@@ -66,6 +70,88 @@ EventRandomVector::EventRandomVector(const RandomVectorImplementation & antecede
   setDescription(antecedent.getDescription());
 }
 
+
+/* Constructor from RandomVector */
+EventRandomVector::EventRandomVector(const RandomVector & antecedent,
+                                     const Interval & interval)
+  : CompositeRandomVector()
+{
+#ifdef OPENTURNS_HAVE_ANALYTICAL_PARSER
```

Comment:
At one point we assume that we always have an analytical parser, otherwise the definition of the bulge function in FORMSystem should also be protected